src/Netscript/NetscriptHelpers.tsx

```diff
@@ -264,12 +264,13 @@ function argsToString(args: unknown[]): string {
       return (out += setToString(nativeArg));
     }
     if (typeof nativeArg === "object") {
-      return (out += JSON.stringify(nativeArg, (_, value) => {
+      return (out += JSON.stringify(nativeArg, (_, value: unknown) => {
         /**
          * If the property is a promise, we will return a string that clearly states that it's a promise object, not a
          * normal object. If we don't do that, all promises will be serialized into "{}".
          */
         if (value instanceof Promise) {
+          // eslint-disable-next-line @typescript-eslint/no-base-to-string -- "[object Promise]" is exactly the string that we want.
           return value.toString();
```

Comment:
Can't we just `return "[object Promise]"` as a static string?

If [object Promise] is the string we want, then let's be explicit, rather than linter suppression and the comment.